helm/regproc-group1/values.yaml

```diff
@@ -78,9 +78,9 @@ startupProbe:
   httpGet:
     path: /registrationprocessor/v1/packetreceiver/actuator/health
     port: 8081
-  initialDelaySeconds: 0
-  periodSeconds: 10
-  timeoutSeconds: 5
+  initialDelaySeconds: 180
+  periodSeconds: 60
+  timeoutSeconds: 30
```

Comment:
For startup probe I thought we are keep the lower side of the number and only increasing the initialDelaySeconds